src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -150,20 +151,19 @@
   <Target Name="GenerateTestBindingRedirects"
           Condition="'$(TestAgainstDesktop)' == 'true' and '$(LocalAppConfigFile)' == ''"
           DependsOnTargets="DiscoverTestInputs;DiscoverTestDependencies">
-    
     <ItemGroup>
-      <_CombinedListOfTestAssets Include="@(IncludedFileForRunnerScript->'%(SourcePath)')" />
-      <_CombinedListOfTestAssets Include="@(SourcesToCopyToTestDir)" />
-      
-      <_FilteredAssembliesForBindingRedirect Include="@(_CombinedListOfTestAssets)" Condition="'%(Extension)'=='.dll' or '%(Extension)'=='.exe'" />
-      <_ExecutablesForBindingRedirect Include="@(SourcesToCopyToTestDir)" Condition="'%(Extension)'!='.pdb'" />
+      <_FilteredAssembliesForBindingRedirect Include="%(_FinalListOfTestAssetsByFileName.SourcePath)" Condition="'%(_FinalListOfTestAssetsByFileName.Extension)'=='.dll' or '%(_FinalListOfTestAssetsByFileName.Extension)'=='.exe'" />
```

Comment:
@karajas It doesn't look like this code was merged correctly; the item type was renamed from `_FinalListOfTestAssetsByFileName` to `_CombinedListOfTestAssets`, and the SourcePath metadata is no longer needed.